draft-ietf-quic-qlog-main-schema.md

```diff
@@ -2026,14 +1843,144 @@ TODO: primarily the .well-known URI
   usage (#26,#33,#51)
 * Overall tightened up the text and added more examples
 
-
 ## Since draft-marx-qlog-main-schema-00:
 
 * All field names are now lowercase (e.g., category instead of CATEGORY)
 * Triggers are now properties on the "data" field value, instead of separate field
   types (#23)
 * group_ids in common_fields is now just also group_id
 
+# Survey of serialization optimizations {#optimization-survey}
+## Data structure optimizations {#structure-optimizations}
```

Comment:
I feel this one can be removed... it's unlikely people will return to this and I don't see much historical value having it in the RFC rather than people looking it up in earlier versions of the drafts themselves.

Reply:
Sure